<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>860</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>KeywordNews</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../../../Desktop/newspaper.png</normaloff>../../../../Desktop/newspaper.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QPushButton" name="num_pushButton_1">
   <property name="geometry">
    <rect>
     <x>42</x>
     <y>150</y>
     <width>230</width>
     <height>80</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 20pt &quot;나눔스퀘어&quot;;
background-color: rgb(227, 227, 255);</string>
   </property>
   <property name="text">
    <string>전체 : 1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="num_pushButton_2">
   <property name="geometry">
    <rect>
     <x>315</x>
     <y>150</y>
     <width>230</width>
     <height>80</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 20pt &quot;나눔스퀘어&quot;;
background-color: rgb(227, 227, 255);</string>
   </property>
   <property name="text">
    <string>정치 : 2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="num_pushButton_3">
   <property name="geometry">
    <rect>
     <x>587</x>
     <y>150</y>
     <width>230</width>
     <height>80</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 20pt &quot;나눔스퀘어&quot;;
background-color: rgb(227, 227, 255);</string>
   </property>
   <property name="text">
    <string>경제 : 3</string>
   </property>
  </widget>
  <widget class="QPushButton" name="num_pushButton_4">
   <property name="geometry">
    <rect>
     <x>42</x>
     <y>280</y>
     <width>230</width>
     <height>80</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 20pt &quot;나눔스퀘어&quot;;
background-color: rgb(227, 227, 255);</string>
   </property>
   <property name="text">
    <string>사회 : 4</string>
   </property>
  </widget>
  <widget class="QPushButton" name="num_pushButton_5">
   <property name="geometry">
    <rect>
     <x>315</x>
     <y>280</y>
     <width>230</width>
     <height>80</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 20pt &quot;나눔스퀘어&quot;;
background-color: rgb(227, 227, 255);</string>
   </property>
   <property name="text">
    <string>국제 : 5</string>
   </property>
  </widget>
  <widget class="QPushButton" name="num_pushButton_6">
   <property name="geometry">
    <rect>
     <x>587</x>
     <y>280</y>
     <width>230</width>
     <height>80</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 20pt &quot;나눔스퀘어&quot;;
background-color: rgb(227, 227, 255);</string>
   </property>
   <property name="text">
    <string>스포츠 : 6</string>
   </property>
  </widget>
  <widget class="QPushButton" name="num_pushButton_7">
   <property name="geometry">
    <rect>
     <x>42</x>
     <y>410</y>
     <width>230</width>
     <height>80</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 20pt &quot;나눔스퀘어&quot;;
background-color: rgb(227, 227, 255);</string>
   </property>
   <property name="text">
    <string>문화 : 7</string>
   </property>
  </widget>
  <widget class="QPushButton" name="num_pushButton_8">
   <property name="geometry">
    <rect>
     <x>315</x>
     <y>410</y>
     <width>230</width>
     <height>80</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 20pt &quot;나눔스퀘어&quot;;
background-color: rgb(227, 227, 255);</string>
   </property>
   <property name="text">
    <string>연예 : 8</string>
   </property>
  </widget>
  <widget class="QPushButton" name="num_pushButton_9">
   <property name="geometry">
    <rect>
     <x>587</x>
     <y>410</y>
     <width>230</width>
     <height>80</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 20pt &quot;나눔스퀘어&quot;;
background-color: rgb(227, 227, 255);</string>
   </property>
   <property name="text">
    <string>IT : 9</string>
   </property>
  </widget>
  <widget class="QPushButton" name="num_pushButton_0">
   <property name="geometry">
    <rect>
     <x>315</x>
     <y>540</y>
     <width>230</width>
     <height>80</height>
    </rect>
   </property>
   <property name="tabletTracking">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 20pt &quot;나눔스퀘어&quot;;
background-color: rgb(227, 227, 255);
</string>
   </property>
   <property name="text">
    <string>종료 : 0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_kn">
   <property name="geometry">
    <rect>
     <x>45</x>
     <y>40</y>
     <width>771</width>
     <height>95</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Baskerville Old Face</family>
     <pointsize>45</pointsize>
    </font>
   </property>
   <property name="text">
    <string>KEYWORD NEWS</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
